src/main.rs

```diff
@@ -34,16 +34,28 @@ struct Args {
 
 fn load_wordlist(path: String) -> io::Result<String> {
     let path = path::Path::new(&path);
-    let mut file = File::open(path)?;
-    let mut string = String::new();
-    file.read_to_string(&mut string)?;
-    Ok(string)
+    if path.exists() {
+        let mut file = File::open(path)?;
+        let mut string = String::new();
+        file.read_to_string(&mut string)?;
+        Ok(string)
+    } else {
+        println!("Error: wordlist '{}' can not be found. Skipping.", path.to_str().unwrap());
+        Ok(String::new())
+    }
 }
 
 fn load_wordlists(paths: Vec<String>) -> io::Result<Vec<String>> {
-    paths.into_iter()
+    let words = paths.into_iter()
         .map(|p| load_wordlist(p))
-        .collect::<Result<Vec<_>, _>>()
+        .collect::<Result<Vec<_>, _>>();
```

Comment:
This doesn't work properly yet. If `load_wordlist` returns an empty string in the error case, then the collect doesn't result in an `Err`. Or am I wrong?

Example:

```
$ cargo run -- -w foo
    Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `target/debug/xkcdpass -w foo`
Error: wordlist 'foo' can not be found. Skipping.
thread 'main' panicked at 'Invalid offset: 0', ../src/libcore/option.rs:705
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```